routing.rst

```diff
@@ -210,103 +166,181 @@ Symfony provides a handy way to declare localized routes without duplication.
-            ])
-                ->controller([CompanyController::class, 'about'])
+            $routes->add('api_post_show', '/api/posts/{id}')
+                ->controller([BlogApiController::class, 'show'])
+                ->methods(['GET', 'HEAD'])
+            ;
+            $routes->add('api_post_edit', '/api/posts/{id}')
+                ->controller([BlogApiController::class, 'edit'])
+                ->methods(['PUT'])
             ;
         };
 
-When a localized route is matched Symfony automatically knows which locale
-should be used during the request. Defining routes this way also eliminated the
-need for duplicate registration of routes which minimizes the risk for any bugs
-caused by definition inconsistency.
-
 .. tip::
 
-    If the application uses full language + territory locales (e.g. ``fr_FR``,
-    ``fr_BE``), you can use the language part only in your routes (e.g. ``fr``).
-    This prevents having to define multiple paths when you want to use the same
-    route path for locales that share the same language.
+    HTML forms only support ``GET`` and ``POST`` methods. If you're calling a
+    route with a different method from an HTML form, add a hidden field called
+    ``_method`` with the method to use (e.g. ``<input type="hidden" name="_method" value="PUT" />``).
+    If you use :doc:`Symfony forms </forms>` this is done automatically for you.
 
-A common requirement for internationalized applications is to prefix all routes
-with a locale. This can be done by defining a different prefix for each locale
-(and setting an empty prefix for your default locale if you prefer it):
+Debugging Routes
+~~~~~~~~~~~~~~~~
+
+As your application grows, you'll eventually have a *lot* of routes. Symfony
+includes some commands to help you debug routing issues. First, the ``debug:router``
+command lists all your application routes in the same order in which Symfony
+evaluates them:
+
+.. code-block:: terminal
+
+    $ php bin/console debug:router
+
+    ----------------  -------  -------  -----  --------------------------------------------
+    Name              Method   Scheme   Host   Path
+    ----------------  -------  -------  -----  --------------------------------------------
+    homepage          ANY      ANY      ANY    /
+    contact           GET      ANY      ANY    /contact
+    contact_process   POST     ANY      ANY    /contact
+    article_show      ANY      ANY      ANY    /articles/{_locale}/{year}/{title}.{_format}
+    blog              ANY      ANY      ANY    /blog/{page}
+    blog_show         ANY      ANY      ANY    /blog/{slug}
+    ----------------  -------  -------  -----  --------------------------------------------
+
+Pass the name (or part of the name) of some route to this argument to print the
+route details:
+
+.. code-block:: terminal
+
+    $ php bin/console debug:router app_lucky_number
+
+      +-------------+---------------------------------------------------------+
```

Comment:
Seems like this is indented 2 spaces too much (see inconsistency with previous terminal output block).
Same in the block below